shell/platform/android/io/flutter/view/FlutterView.java

```diff
@@ -267,7 +267,7 @@ public void addActivityLifecycleListener(ActivityLifecycleListener listener) {
     }
 
     public void onPause() {
-        mFlutterLifecycleChannel.send("AppLifecycleState.paused");
+        mFlutterLifecycleChannel.send("AppLifecycleState.inactive");
```

Comment:
Might be worth testing, I might be wrong, but I think on most Android manufacturer OSes, the lock screen might put us in this state too. We should be sure about battery usage for common cases.

Reply:
I'll take a look at battery use. I'm fairly certain that Choreographer won't give use frames, so hopefully it looks okay.

Reply:
Did some research - we might be able to tell using the PowerManager API

Reply:
I checked a few devices (pixel 1, nexus 5, galaxy s5) - all will call onStop when exiting to the lock screen.  I think this should be okay as is